locales/en-US/notes.properties

```diff
@@ -123,3 +123,68 @@ newNote=New Note
 makePlainText=Make Plain Text
 deleteNote=Delete Note
 insufficientStorage=Notes Storage Full
+
+
+#
+# Mobile app translations only start
+
+# LOCALIZATION NOTE (welcomeTitle4): This string is the login panel of the mobile app
+welcomeTitle4=Notes by Firefox
+
+# LOCALIZATION NOTE (welcomeHeadline): This string is the login panel of the mobile app
+welcomeHeadline=A simple, secure notepad app that syncs with Firefox
+
+# LOCALIZATION NOTE (signIn): This string is in the button in the login panel of the mobile app
+signIn=Sign In
+
+# LOCALIZATION NOTE (usageHint): This string is the login panel of the mobile app
+usageHint=To use Notes, you’ll need a Firefox Account and the free Notes browser extension from Firefox Test Pilot.
```

Comment:
It's the first time I see "Firefox Test Pilot" instead of just "Test Pilot". Is this correct?

Reply:
True!

These were propose by the copy writer. @johngruen to do final r+